DocumentManagementSystem/settings.py

```
@@ -37,6 +37,7 @@
     os.getenv('ipadd1'),
     os.getenv('ipadd2'),
     os.getenv('ipadd3'),
+    os.getenv('ipadd4'),
```

Ruff (0.8.2)

Use capitalized environment variable `IPADD4` instead of `ipadd4`

(SIM112)